index.html

```diff
@@ -56,19 +56,19 @@
         <div class="col-md-4">
             <i class="top3-icon fa fa-thumbs-up"></i>
 
-            <p class="strong-heading">Simple to use</p>
+            <p class="strong-heading">Simple to</br> use</p>
```

Comment:
This feels a bit hacky and is still is funky when the site is responsive - there's a neater way to align text across rows with flexbox/css.

Reply:
Done in latest commit